tests/test_tf.py

```diff
@@ -116,25 +116,6 @@ def circuit(x):
         assert isinstance(res, tf.Tensor)
         assert np.allclose(res, 1, atol=tol, rtol=0)
 
-    def test_nonzero_shots(self):

```

Comment:
An error is being raised when `diff_method=backprop` and `shots!=None` is being specified as per https://github.com/PennyLaneAI/pennylane/pull/1588.

This test case is now outdated.